docs/make.jl

```diff
@@ -31,6 +31,7 @@ examples = [
     "One-dimensional diffusion"        => "one_dimensional_diffusion",
     "Two-dimensional turbulence"       => "two_dimensional_turbulence",
     "Internal wave"                    => "internal_wave",
+    "Barotropic tide"                  => "barotropic_tide",
```

Comment:
I don't think "barotropic tide" is a good name for this example. People typically refer to these motions as "internal tide" (which contrasts barotropic tide; ie the internal tide consists of _baroclinic_ motions at the tidal frequency, whereas the barotropic tide is purely barotropic).

Reply:
agreed!

Reply:
see my comment below, which is better title?

internal_tide.jl + "Internal tide generation from a seamount" ?

Reply:
where below?

Reply:
I thikn that sound good, but "by a seamount" not "from a seamount"